src/liballoc/tests/btree/map.rs

```diff
@@ -578,14 +579,15 @@ fn test_range() {
 
 #[test]
 fn test_range_mut() {
+    let size = 199;
```

Comment:
Used to be 200, now 199... huh?

Reply:
Just so that the maximum size 198 is almost the same as the 199 before and is reached by steps of 66. Confusing, I agree. Trying to minimize changes and promote the use of Miri (it's not "too slow", it just doesn't do crazy work).

Reply:
There'd be no harm in leaving it at 200 though, right? So that would "minimize changes" even more.
Not that it matters a lot. Just wondering.

Reply:
No harm indeed. It merely continues to feed Miri a range including the very last element of the map. But this test case doesn't cover many other kinds of ranges, and that's why I adapted and added test cases a while back, which Miri executes happily. Most likely, this test case is redundant anyways.